src/openrct2/paint/VirtualFloor.cpp

```diff
@@ -139,6 +131,14 @@ void virtual_floor_paint(paint_session * session)
     //  if our parameters differ we set an edge towards that tile
     for (uint8 i = 0; i < 4; i++)
     {
+        static constexpr const CoordsXY offsets[4] =
```

Comment:
Having this in a loop now seems really strange to me.

Reply:
Yes, can you at least move them out of the loop? Also for legibility reasons.